payments/billing-sdk/src/main/java/com/appcoins/billing/sdk/BillingErrorMapper.kt

```diff
@@ -12,8 +10,7 @@ internal interface BillingErrorMapper {
   fun mapConsumePurchasesError(exception: Throwable): Int
 }
 
-@Singleton
-internal class BillingErrorMapperImpl @Inject constructor() : BillingErrorMapper {
+internal class BillingErrorMapperImpl internal constructor() : BillingErrorMapper {
```

Comment:
The internal class has internal constructor by default so we don't need to specify the `internal` modifier for it explicitly